.github/workflows/publish-package.yml

```diff
@@ -18,8 +18,17 @@ jobs:
           scope: '@eclipse-velocitas'
       - run: npm ci
       - run: npm run build
+      - run: npm run coverage
```

Comment:
is coverage and test needed? or is coverage already executing the tests? please check this

Reply:
@mml5bg did you test that the tests are executed correctly by adding a failing test e.g.?

Reply:
If you open the workflow, it is visible that is executed, there is no need to add failing test.

Reply:
but is the workflow also failing if a tests fails? this is still unclear to me if we wrap it with nyc...

Reply:
That is unclear, that needs to tested

Reply:
As expected, it is failing.